Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/display/FhirDisplayController.java

```diff
@@ -41,6 +41,7 @@
 public class FhirDisplayController implements DisplayController {
 
     private static final Logger LOG = LoggerFactory.getLogger(FhirDisplayController.class);
+    private static final WebServiceProxyHelper PROXY_HELPER = new WebServiceProxyHelper();
```

Comment:
Optional: don't use upper case for variable.  Only use upper case for constant variable instead.